docs/templates/extension.md

```diff
@@ -0,0 +1,46 @@
+# Title of the Extension
+
+_Briefly describe the functionality this extension introduces._
+
+## Background
+
+_Briefly describe the relevance and need of this extension._
+
+### Scope
+
+_Describe what this extension should be used for, e.g., for what kind of data transfers or data
+sharing scenarios._
+
+### Use Cases
+
+_Explain one or multiple use cases that could be realized using this extension._
+
+## Technical Details
+
+### Interfaces
+
+_Provide a detailed description of all provided and consumed interfaces._
```

Comment:
sufficient to point to appropriate classes? wonder what level of detail is sufficient...

Reply:
Added some more sentences